common/app/common/configuration.scala

```diff
@@ -651,6 +651,8 @@ class GuardianConfiguration extends GuLogging {
 
     lazy val region = configuration.getMandatoryStringProperty("aws.region")
     lazy val frontendStoreBucket = configuration.getMandatoryStringProperty("aws.bucket")
+    lazy val topMentionsStoreBucket =
+      configuration.getMandatoryStringProperty("aws.topMentions.bucket") // TODO: Add this to ssm
```

Comment:
Is this mandatory? Could the app just run with the top mentions service disabled if the property isn't there and log something out?

Reply:
Thanks so much for spotting this. It's certainly shouldn't be mandatory as we don't want the service to go down if this parameter doesn't exist. 

I made it optional, and the method in `TopMentionsS3ClientImpl` are now resulting in Future.failed if the parameter doesn't have value.